src/components/Page.tsx

```diff
@@ -0,0 +1,15 @@
+import styled from "styled-components";
+
+const PageWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100vw`;
+
+// @ts-ignore
+export const Page = (props) => {
```

Comment:
the better way how to handle children props is like this:
1. create a type (or interface) before the function
2. use the props type (or interface) as a function's argument type
3. use object destructuring (optional) to take the separate arguments

```
interface Props {
  children: React.ReactNode;
}

const Page = ({children}: Props) => {
return (
  <>
    {children}
  </>
);
}
```